packet/baseapi.py

```diff
@@ -47,12 +49,21 @@ class BaseAPI(object):
         Basic api class for
     """
 
-    def __init__(self, auth_token, consumer_token):
+    def __init__(self, auth_token, consumer_token, user_agent=None):
```

Comment:
If you change the default case to

```suggestion
    def __init__(self, auth_token, consumer_token, user_agent=""):
```

then you don't need the nested format call below.